src/components/classItem/classItem.style.scss

```
@@ -0,0 +1,111 @@
+@use "../../style/variables" as var;
+@use "../../style/mixins.scss" as mixin;
+
+.class-item {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 4px;
+  padding: 8px 4px;
+  width: 72px;
+  height: 64px;
+  text-align: center;
+  cursor: pointer;
+
+  &--empty {
+    cursor: default;
+  }
+
+  &--default {
+    background-color: var.$blue-darker;
+    border: 1px solid var.$divider-color;
+    &:hover {
+      box-shadow: 0px 0px 2px 0px var.$blue-weak-emphasis;
+    }
+    &:active {
+      box-shadow: none;
+      opacity: 80%;
+    }
+  }
+
+  &--disabled {
+    background-color: var.$black;
+    border: 1px dashed var.$divider-color;
+    &:hover {
+      border-color: var.$blue-weak-emphasis;
+    }
+    &:active {
+      border-color: var.$divider-color;
+      opacity: 80%;
+    }
+  }
+
+  &--selected {
+    background-color: var.$blue-darker;
+    border: 1px solid var.$blue-gray;
+    &:hover {
+      box-shadow: 0px 0px 2px 0px var.$blue-weak-emphasis;
+    }
+    &:active {
+      box-shadow: none;
+      opacity: 80%;
+    }
+  }
+
+  &__checkbox {
+    position: absolute;
+    top: 5.1px;
+    right: 5.1px;
+    width: 8px;
+    height: 8px;
+    border-radius: 1px;
```

Border radius deve ser de 2px

```suggestion
    border-radius: 2px;
```